packages/fast-components/src/anchored-region/anchored-region.template.ts

```diff
@@ -0,0 +1,14 @@
+import { html, ref, when } from "@microsoft/fast-element";
+import { AnchoredRegion } from "./anchored-region";
+
+export const AnchoredRegionTemplate = html<AnchoredRegion>`
+  <template>
```

Comment:
You don't need the template tag here if you're not adding any attributes to it.

Reply:
apparently I don't